content/docs/start/metrics-parameters-plots.md

```diff
@@ -271,5 +274,5 @@ file:///Users/dvc/example-get-started/plots.html
 > [Git revisions](https://git-scm.com/docs/gitrevisions) (commits, tags, branch
 > names) to compare.
 
-In the next page, learn advanced ways to track, organize, and compare more
-experiment iterations.
+On the next page, you can learn advanced ways to track, organize, and compare
```

Comment:
"In" was right?

Reply:
No, you find things on a website, on a sign, in a book, on a page